internal/opa/output.go

```diff
@@ -83,13 +85,22 @@ func prepTemplateFields(a *ast.AnnotationsRef) TemplateFields {
 		// conventions i.e. "package policy.<type>.<name>"
 		shortPath = trimmedPath
 	}
+
+	shortName := getShortName(a)
+
+	var docsUrl string
+	if len(pathStrings) > 4 && shortName != "" && (strings.HasPrefix(trimmedPath, "policy.pipeline.") || strings.HasPrefix(trimmedPath, "policy.release.")) {
+		docsUrl = fmt.Sprintf("https://hacbs-contract.github.io/ec-policies/%s_policy.html#%s__%s", pathStrings[2], pathStrings[3], shortName)
```

Comment:
Later we could have a package annotation with the default docs url prefix. For now - hard coded.